content/docs/0.17.x/reference/cli/commands/keptn_send.md

```diff
@@ -13,7 +13,7 @@ Sends an event to Keptn
       --config-file string   Specify custom Keptn Config file path (default: ~/.keptn/config)
   -h, --help                 help
       --mock                 Disables communication to a Keptn endpoint
-  -n, --namespace string     Specify the namespace where Keptn should be installed, used and uninstalled in (default "keptn")
+  -n, --namespace string     Specify the namespace where Keptn should be installed, used and uninstalled (default "keptn")
```

Comment:
Should this be "where Keptn is installed" or "where the Keptn Helm chart is installed"?  "should be" implies that this command will do that installation.